subjects/gatecrashers/README.md

```diff
@@ -0,0 +1,64 @@
+## Gatecrashers
+
+### Instructions
+
+Unfortunately many of your guests started to invite plus ones, that started to invite plus ones, that started... to be short, everybody is inviting everybody and the situation is rapidly going out of control.
+
+To fix this issue you will introduce a [Basic Access Authentication](https://en.wikipedia.org/wiki/Basic_access_authentication) on your server.
+
+To modify the guest list now your friends will need to make authenticated requests. You decided that only your 3 best friends (`Caleb_Squires`, `Tyrique_Dalton` and `Rahima_Young`) will be able to modify the list. You also told them the secret password `abracadabra` in order to do that.
+
+The request query will look like this `curl -u Caleb_Squires:abracadabra ...`.
+
+The server should properly handle unauthorized requests using the error code `401`.
+
+Apart for the authentication part your server's behavior should remain unchanged:
+- You will listen to port `5000` and print a message containing the port you are using.
+- For each **authorized** http `POST` request, your program should create the corresponding JSON file and store the contents of the body, and then provide the content as JSON in the HTTP response, if possible. 
+
```

Comment:
For consistency this should be specified in the previous exercise (this info is missing at the moment), we could wait for that.
Also we have "your server's behavior should remain unchanged" which will hopefully be enough to point the student in the right direction.

Reply:
I would still add a very short sentence `the server will be tested only to add new guest - no already existing name will be tested` or something similar to avoid doubts

Reply:
I've explicitly quoted the previous exercise as discussed to give more hints about how the server should behave